content/300-guides/500-other/880-troubleshooting-orm/050-creating-bug-reports.mdx

```diff
@@ -120,6 +120,18 @@ Please include any information about your environment and setup. Specifically it
 - Which **version of Prisma** you use (run `prisma -v` to see your Prisma version)
 - Which **version of Node.js** you use (run `node -v` to see your Node.js version)
 
+You can use the [`prisma debug`](/reference/api-reference/command-reference#debug) <span class="api"></span> command to retrieve debugging information.
```

Comment:
When do I want to use this vs. what is described above?

Reply:
I've added more information. Let me know if it reads better.